proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -17,11 +17,11 @@ message BaseAccount {
 
   option (cosmos_proto.implements_interface) = "AccountI";
 
-  string              address = 1;
+  string              address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
   google.protobuf.Any pub_key = 2
       [(gogoproto.jsontag) = "public_key,omitempty", (gogoproto.moretags) = "yaml:\"public_key\""];
-  uint64 account_number = 3 [(gogoproto.moretags) = "yaml:\"account_number\""];
-  uint64 sequence       = 4;
+  uint64 account_number = 3 [(gogoproto.moretags) = "yaml:\"account_number\""], [(cosmos_proto.scalar) = "cosmos.Int"];
+  uint64 sequence       = 4 [(cosmos_proto.scalar) = "cosmos.Int"];
```

Comment:
just wondering if I may apply `cosmos.Int` annotation to `uint64` Go type

Reply:
No, just string types